tests/acceptance/shared_browser/hold_open_test.py

```diff
@@ -0,0 +1,23 @@
+from selene import have
+from selene.support.shared import config, browser, SharedConfig
+
+
+def setup_module():
+    config.hold_browser_open = True
```

Comment:
hold_browser_open_on_session_exit - this is what is meant under hold_browser_open....

Reply:
that's why test cases below do not actually test this config option....

Reply:
But look at the tests.
They test if browser opened through one test session. That's covers the scenario.

But you are right. Scenario with browser open after pytest session will be nice to have ;)
I think how to achieve that test, and my thoughts are: somehow get id of webdriver and browser processes in pytest session, write them down to the file in the end of session. And after session execute python script which will check system processes trying to find these ids... and if they will be found - test will pass, and thats will be the test you meant.